### ISSUE TITLE
Placing Live Bids

### ISSUE THREAD

**Issue by ashfurrow** [artsy/eigen#1109]
[Place a bid](https://github.com/artsy/eigen/issues/1081) flow for placing bid and displaying results of bid (only instance where we send a WebSocket event and need to wait for a corresponding event)

**Comment by ashfurrow** [comment]
Estimate 1 Day.

**Comment by devangt** [comment]
Waiting on deployment and real data

cc @acjay

**Comment by katarinabatina** [comment]
working somewhat but the [ BIDDING... ] state that the button gets into is inaccurate as the bidder should go from bid sent, to winning and or outbid, and then be allowed to bid again.

**Comment by ashfurrow** [comment]
Hmm, that makes sense. Let's put those changes in a separate issue? I'd really like to close this showstopper :wink:
